generators/entity-server/templates/src/main/java/package/common/inject_template.ejs

```diff
@@ -39,7 +39,7 @@
     }
 _%>
 
-    <%= beans.map(bean => `private final ${bean.class} ${bean.instance};`).join('\n\n    ') %>
+    <%= beans.map(bean => `protected final ${bean.class} ${bean.instance};`).join('\n\n    ') %>
```

Comment:
This is from another commit, and I know we must discuss this, but we can't merge this with this PR anyway

Reply:
I'm correcting this and trying to merge....